pkgs/servers/headphones/default.nix

```diff
@@ -0,0 +1,34 @@
+{ stdenv, fetchFromGitHub, python2, makeWrapper }:
+
+python2.pkgs.buildPythonApplication rec {
+  name = "headphones-${version}";
+  version = "0.5.19";
+
+  src = fetchFromGitHub {
+    owner = "rembo10";
+    repo = "headphones";
+    rev = "v${version}";
+    sha256 = "0z39gyan3ksdhnjxxs7byamrzmrk8cn15g300iqigzvgidff1lq0";
+  };
+
+  dontBuild = true;
+  doCheck = false;
+
+  nativeBuildInputs = [ makeWrapper ];
+  buildInputs = [ python2 ];
+
+  installPhase = ''
```

Comment:
Could you also remove the files that wouldn't be useful and nix and stuff like the `README.md`?

Reply:
Something like this?
```
    installPhase = ''
        mkdir -p $out/bin
        cp -R {data,headphones,lib,Headphones.py} $out/  
        makeWrapper $out/Headphones.py $out/bin/headphones
    ''
```

Do I need the license file?

Reply:
Yeah just what's essential :+1: 

> Do I need the license file?

No, the meta attribute exposes that information.

Reply:
Cool - that's what I thought. Do you think it would be useful to include the init scritps? In case someone is using on something other than nixos?

Reply:
Not likely as they reference files that would be in the nix store so you'd have to patch all of them.
But if you did they should go to `$out/bin` since they're utility scripts.

Reply:
Ok - I'm alright with not including them. Just one last thing - I had the `echo "v${version}" >> $out/version.txt` line because headphones checks that file to see what version it's on for the updater script, otherwise it reports that it's running an unknown version. Is it bad practice to include something like that?

Reply:
As long as the updater script silently fails because the nix store is read only and cannot be mutated.

So having  `echo "v${version}" >> $out/version.txt` wouldn't matter because it would fail anyway.

> Is it bad practice to include something like that?

I would say that it's best to integrate with any of the various packaging systems in their respective platforms.